views/player/partials/giveItemButton.ejs

```diff
@@ -96,7 +97,7 @@
       let quality = $('#player-give-item-input-quality').val()
 
       if (!(amount == '')) {
-        if (isNaN(amount)) {
+        if (isNaN(quality)) {
```

Comment:
Ran this locally and noticed a bug with this. `.val()` returns a string. Which means `isNaN(quality)` will never work.

Either we should `parseInt(quality, 10)` or adjust the check to account for empty strings.

 `let quality = parseInt($('#player-give-item-input-quality').val(), 10)`

Wanted to push but no rights to the forked repo :smile:

Reply:
I've updated with your suggestion; should we also do the same thing with amount?

Reply:
amount already has a check for empty strings, should be fine as is! I'll give the new version another test later today and merge, should be good now I think